CI: LVH VM startup fails with exit code 41

## CI failure

```
[    0.000000] Linux version 5.4.268 (root@buildkitsandbox) (gcc version 13.2.0 (Ubuntu 13.2.0-4ubuntu3)) #1 SMP Thu Feb 15 09:40:20 UTC 2024
[    0.000000] Command line: BOOT_IMAGE=/vmlinuz initrd=initrd.img root=/dev/vda rw console=ttyS0
[    0.000000] BIOS-provided physical RAM map:
[    0.000000] BIOS-e820: [mem 0x0000000000000000-0x000000000009fbff] usable
[    0.000000] BIOS-e820: [mem 0x000000000009fc00-0x000000000009ffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000000f0000-0x00000000000fffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000000100000-0x00000000bffdcfff] usable
[    0.000000] BIOS-e820: [mem 0x00000000bffdd000-0x00000000bfffffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000feffc000-0x00000000feffffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000fffc0000-0x00000000ffffffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000100000000-0x000000033fffffff] usable
[    0.000000] NX (Execute Disable) protection: active
[    0.000000] SMBIOS 2.8 present.
[    0.000000] DMI: QEMU Standard PC (i440FX + PIIX, 1996), BIOS 1.15.0-1 04/01/2014
[    0.000000] Hypervisor detected: KVM
[    0.000000] kvm-clock: Using msrs 4b564d01 and 4b564d00
[    0.000000] kvm-clock: cpu 0, msr 32b759001, primary cpu clock
[    0.000000] kvm-clock: using sched offset of 417358316 cycles
[    0.000001] clocksource: kvm-clock: mask: 0xffffffffffffffff max_cycles: 0x1cd42e4dffb, max_idle_ns: 881590591483 ns
[    0.000003] tsc: Detected 2445.432 MHz processor
[    0.000365] last_pfn = 0x340000 max_arch_pfn = 0x400000000
[    0.000560] x86/PAT: Configuration [0-7]: WB  WC  UC- UC  WB  WP  UC- WT  
[    0.000596] last_pfn = 0xbffdd max_arch_pfn = 0x400000000
[    0.002450] found SMP MP-table at [mem 0x000f5b90-0x000f5b9f]
[    0.002491] check: Scanning 1 areas for low memory corruption
[    0.002540] Using GB pages for direct mapping
[    0.002663] RAMDISK: [mem 0x7d9e0000-0x7fffffff]
[    0.002668] ACPI: Early table checksum verification disabled
[    0.002681] ACPI: RSDP 0x00000000000F59B0 000014 (v00 BOCHS )
[    0.002686] ACPI: RSDT 0x00000000BFFE19C8 000034 (v01 BOCHS  BXPC     00000001 BXPC 00000001)
[    0.002689] ACPI: FACP 0x00000000BFFE1864 000074 (v01 BOCHS  BXPC     00000001 BXPC 00000001)
[    0.002691] ACPI: DSDT 0x00000000BFFE0040 001824 (v01 BOCHS  BXPC     00000001 BXPC 00000001)
[    0.002693] ACPI: FACS 0x00000000BFFE0000 000040
[    0.002694] ACPI: APIC 0x00000000BFFE18D8 000090 (v01 BOCHS  BXPC     00000001 BXPC 00000001)
[    0.002696] ACPI: HPET 0x00000000BFFE1968 000038 (v01 BOCHS  BXPC     00000001 BXPC 00000001)
[    0.002697] ACPI: WAET 0x00000000BFFE19A0 000028 (v01 BOCHS  BXPC     00000001 BXPC 00000001)
[    0.002699] ACPI: Reserving FACP table memory at [mem 0xbffe1864-0xbffe18d7]
[    0.002699] ACPI: Reserving DSDT table memory at [mem 0xbffe0040-0xbffe1863]
[    0.002700] ACPI: Reserving FACS table memory at [mem 0xbffe0000-0xbffe003f]
[    0.002700] ACPI: Reserving APIC table memory at [mem 0xbffe18d8-0xbffe1967]
[    0.002701] ACPI: Reserving HPET table memory at [mem 0xbffe1968-0xbffe199f]
[    0.002701] ACPI: Reserving WAET table memory at [mem 0xbffe19a0-0xbffe19c7]
[    0.003275] No NUMA configuration found
[    0.003276] Faking a node at [mem 0x0000000000000000-0x000000033fffffff]
[    0.003278] NODE_DATA(0) allocated [mem 0x33fffc000-0x33fffffff]
[    0.003646] Zone ranges:
[    0.003647]   DMA      [mem 0x0000000000001000-0x0000000000ffffff]
[    0.003648]   DMA32    [mem 0x0000000001000000-0x00000000ffffffff]
[    0.003649]   Normal   [mem 0x0000000100000000-0x000000033fffffff]
[    0.003649] Movable zone start for each node
[    0.003650] Early memory node ranges
[    0.003650]   node   0: [mem 0x0000000000001000-0x000000000009efff]
[    0.003651]   node   0: [mem 0x0000000000100000-0x00000000bffdcfff]
[    0.003652]   node   0: [mem 0x0000000100000000-0x000000033fffffff]
[    0.004009] Zeroed struct page in unavailable ranges: 133 pages
[    0.004010] Initmem setup node 0 [mem 0x0000000000001000-0x000000033fffffff]
[    0.044599] ACPI: PM-Timer IO Port: 0x608
[    0.044607] ACPI: LAPIC_NMI (acpi_id[0xff] dfl dfl lint[0x1])
[    0.044675] IOAPIC[0]: apic_id 0, version 17, address 0xfec00000, GSI 0-23
[    0.044677] ACPI: INT_SRC_OVR (bus 0 bus_irq 0 global_irq 2 dfl dfl)
[    0.044678] ACPI: INT_SRC_OVR (bus 0 bus_irq 5 global_irq 5 high level)
[    0.044679] ACPI: INT_SRC_OVR (bus 0 bus_irq 9 global_irq 9 high level)
[    0.044679] ACPI: INT_SRC_OVR (bus 0 bus_irq 10 global_irq 10 high level)
[    0.044680] ACPI: INT_SRC_OVR (bus 0 bus_irq 11 global_irq 11 high level)
[    0.044683] Using ACPI (MADT) for SMP configuration information
[    0.044684] ACPI: HPET id: 0x8086a201 base: 0xfed00000
[    0.044687] TSC deadline timer available
[    0.044695] smpboot: Allowing 4 CPUs, 0 hotplug CPUs
[    0.044736] KVM setup pv sched yield
[    0.044742] PM: Registered nosave memory: [mem 0x00000000-0x00000fff]
[    0.044743] PM: Registered nosave memory: [mem 0x0009f000-0x0009ffff]
[    0.044744] PM: Registered nosave memory: [mem 0x000a0000-0x000effff]
[    0.044744] PM: Registered nosave memory: [mem 0x000f0000-0x000fffff]
[    0.044745] PM: Registered nosave memory: [mem 0xbffdd000-0xbfffffff]
[    0.044745] PM: Registered nosave memory: [mem 0xc0000000-0xfeffbfff]
[    0.044746] PM: Registered nosave memory: [mem 0xfeffc000-0xfeffffff]
[    0.044746] PM: Registered nosave memory: [mem 0xff000000-0xfffbffff]
[    0.044747] PM: Registered nosave memory: [mem 0xfffc0000-0xffffffff]
[    0.044748] [mem 0xc0000000-0xfeffbfff] available for PCI devices
[    0.044749] Booting paravirtualized kernel on KVM
[    0.044751] clocksource: refined-jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 1910969940391419 ns
[    0.128122] setup_percpu: NR_CPUS:64 nr_cpumask_bits:64 nr_cpu_ids:4 nr_node_ids:1
[    0.128380] percpu: Embedded 56 pages/cpu s191192 r8192 d29992 u524288
[    0.128435] KVM setup async PF for cpu 0
[    0.128472] kvm-stealtime: cpu 0, msr 333a1b1c0
[    0.128475] Built 1 zonelists, mobility grouping on.  Total pages: 3096422
[    0.128476] Policy zone: Normal
[    0.128477] Kernel command line: BOOT_IMAGE=/vmlinuz initrd=initrd.img root=/dev/vda rw console=ttyS0
[    0.130387] Dentry cache hash table entries: 2097152 (order: 12, 16777216 bytes, linear)
[    0.131301] Inode-cache hash table entries: 1048576 (order: 11, 8388608 bytes, linear)
[    0.131355] mem auto-init: stack:off, heap alloc:off, heap free:off
[    0.151727] Memory: 12224416K/12582380K available (12291K kernel code, 1359K rwdata, 6348K rodata, 1588K init, 1532K bss, 357964K reserved, 0K cma-reserved)
[    0.151963] SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
[    0.151974] ftrace: allocating 36525 entries in 143 pages
[    0.163260] ftrace: allocated 143 pages with 5 groups
[    0.163490] rcu: Hierarchical RCU implementation.
[    0.163491] rcu: 	RCU event tracing is enabled.
[    0.163491] rcu: 	RCU restricting CPUs from NR_CPUS=64 to nr_cpu_ids=4.
[    0.163493] rcu: RCU calculated value of scheduler-enlistment delay is 100 jiffies.
[    0.163494] rcu: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
[    0.165285] NR_IRQS: 4352, nr_irqs: 456, preallocated irqs: 16
[    0.165512] random: crng init done
[    0.165681] Console: colour *CGA 80x25
[    0.334416] printk: console [ttyS0] enabled
[    0.335526] ACPI: Core revision 20190816
[    0.336770] clocksource: hpet: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604467 ns
[    0.339289] APIC: Switch to symmetric I/O mode setup
[    0.340671] KVM setup pv IPIs
[    0.344129] ..TIMER: vector=0x30 apic1=0 pin1=2 apic2=-1 pin2=-1
[    0.345824] tsc: Marking TSC unstable due to TSCs unsynchronized
[    0.347491] Calibrating delay loop (skipped) preset value.. 4890.86 BogoMIPS (lpj=2445432)
[    0.348739] x86/cpu: User Mode Instruction Prevention (UMIP) activated
[    0.349738] Last level iTLB entries: 4KB 512, 2MB 255, 4MB 127
[    0.350490] Last level dTLB entries: 4KB 512, 2MB 255, 4MB 127, 1GB 0
[    0.351497] Spectre V1 : Mitigation: usercopy/swapgs barriers and __user pointer sanitization
[    0.352493] Spectre V2 : Mitigation: Retpolines
[    0.353490] Spectre V2 : Spectre v2 / SpectreRSB mitigation: Filling RSB on context switch
[    0.354490] Spectre V2 : Spectre v2 / SpectreRSB : Filling RSB on VMEXIT
[    0.355490] Speculative Store Bypass: Vulnerable
[    0.356542] x86/fpu: Supporting XSAVE feature 0x001: 'x87 floating point registers'
[    0.357490] x86/fpu: Supporting XSAVE feature 0x002: 'SSE registers'
[    0.358490] x86/fpu: Supporting XSAVE feature 0x004: 'AVX registers'
[    0.359518] x86/fpu: xstate_offset[2]:  576, xstate_sizes[2]:  256
[    0.360490] x86/fpu: Enabled xstate features 0x7, context size is 832 bytes, using 'compacted' format.
[    0.362558] Freeing SMP alternatives memory: 36K
[    0.363497] pid_max: default: 32768 minimum: 301
[    0.364504] LSM: Security Framework initializing
[    0.365495] SELinux:  Initializing.
[    0.366508] Mount-cache hash table entries: 32768 (order: 6, 262144 bytes, linear)
[    0.367498] Mountpoint-cache hash table entries: 32768 (order: 6, 262144 bytes, linear)
[    0.368941] smpboot: CPU0: AMD EPYC 7763 64-Core Processor (family: 0x19, model: 0x1, stepping: 0x1)
[    0.369548] Performance Events: PMU not available due to virtualization, using software events only.
[    0.370570] rcu: Hierarchical SRCU implementation.
[    0.371520] Huh? What family is it: 0x19?!
[    0.372523] smp: Bringing up secondary CPUs ...
[    0.373589] x86: Booting SMP configuration:
[    0.374498] .... node  #0, CPUs:      #1
[    0.178992] kvm-clock: cpu 1, msr 32b759041, secondary cpu clock
[    0.377340] KVM setup async PF for cpu 1
[    0.377489] kvm-stealtime: cpu 1, msr 333a9b1c0
[    0.380598]  #2
[    0.178992] kvm-clock: cpu 2, msr 32b759081, secondary cpu clock
[    0.383249] KVM setup async PF for cpu 2
[    0.383489] kvm-stealtime: cpu 2, msr 333b1b1c0
[    0.386607]  #3
[    0.178992] kvm-clock: cpu 3, msr 32b7590c1, secondary cpu clock
[    0.389072] KVM setup async PF for cpu 3
[    0.389489] kvm-stealtime: cpu 3, msr 333b9b1c0
[    0.392594] smp: Brought up 1 node, 4 CPUs
[    0.393498] smpboot: Max logical packages: 1
[    0.394496] smpboot: Total of 4 processors activated (19563.45 BogoMIPS)
[    0.396762] devtmpfs: initialized
[    0.397576] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 1911260446275000 ns
[    0.398510] futex hash table entries: 1024 (order: 4, 65536 bytes, linear)
[    0.399647] PM: RTC time: 07:11:42, date: 2024-03-07
[    0.400657] NET: Registered protocol family 16
[    0.401536] audit: initializing netlink subsys (disabled)
[    0.402513] audit: type=2000 audit(1709795503.440:1): state=initialized audit_enabled=0 res=1
[    0.404497] cpuidle: using governor menu
[    0.407614] KVM setup pv remote TLB flush
[    0.408497] ACPI: bus type PCI registered
[    0.409636] PCI: Using configuration type 1 for base access
[    0.410497] PCI: Using configuration type 1 for extended access
[    0.412530] HugeTLB registered 2.00 MiB page size, pre-allocated 0 pages
[    0.413539] cryptomgr_test (35) used greatest stack depth: 15384 bytes left
[    0.416534] ACPI: Added _OSI(Module Device)
[    0.417497] ACPI: Added _OSI(Processor Device)
[    0.418497] ACPI: Added _OSI(3.0 _SCP Extensions)
[    0.420498] ACPI: Added _OSI(Processor Aggregator Device)
[    0.421496] ACPI: Added _OSI(Linux-Dell-Video)
[    0.422498] ACPI: Added _OSI(Linux-Lenovo-NV-HDMI-Audio)
[    0.424498] ACPI: Added _OSI(Linux-HPI-Hybrid-Graphics)
[    0.425892] ACPI: 1 ACPI AML tables successfully acquired and loaded
[    0.428530] ACPI: Interpreter enabled
[    0.429506] ACPI: (supports S0 S3 S4 S5)
[    0.430497] ACPI: Using IOAPIC for interrupt routing
[    0.432506] PCI: Using host bridge windows from ACPI; if necessary, use "pci=nocrs" and report a bug
[    0.434573] ACPI: Enabled 2 GPEs in block 00 to 0F
[    0.437892] ACPI: PCI Root Bridge [PCI0] (domain 0000 [bus 00-ff])
[    0.439501] acpi PNP0A03:00: _OSC: OS supports [ExtendedConfig ASPM ClockPM Segments MSI HPX-Type3]
[    0.441524] PCI host bridge to bus 0000:00
[    0.442498] pci_bus 0000:00: root bus resource [io  0x0000-0x0cf7 window]
[    0.444497] pci_bus 0000:00: root bus resource [io  0x0d00-0xffff window]
[    0.446498] pci_bus 0000:00: root bus resource [mem 0x000a0000-0x000bffff window]
[    0.448496] pci_bus 0000:00: root bus resource [mem 0xc0000000-0xfebfffff window]
[    0.450498] pci_bus 0000:00: root bus resource [mem 0x340000000-0x3bfffffff window]
[    0.452496] pci_bus 0000:00: root bus resource [bus 00-ff]
[    0.454586] pci 0000:00:00.0: [8086:1237] type 00 class 0x060000
[    0.456675] pci 0000:00:01.0: [8086:7000] type 00 class 0x060100
[    0.459686] pci 0000:00:01.1: [8086:7010] type 00 class 0x010180
[    0.464374] pci 0000:00:01.1: reg 0x20: [io  0xc0e0-0xc0ef]
[    0.467549] pci 0000:00:01.1: legacy IDE quirk: reg 0x10: [io  0x01f0-0x01f7]
[    0.469499] pci 0000:00:01.1: legacy IDE quirk: reg 0x14: [io  0x03f6]
[    0.470498] pci 0000:00:01.1: legacy IDE quirk: reg 0x18: [io  0x0170-0x0177]
[    0.472497] pci 0000:00:01.1: legacy IDE quirk: reg 0x1c: [io  0x0376]
[    0.474749] pci 0000:00:01.3: [8086:7113] type 00 class 0x068000
[    0.477518] pci 0000:00:01.3: quirk: [io  0x0600-0x063f] claimed by PIIX4 ACPI
[    0.479516] pci 0000:00:01.3: quirk: [io  0x0700-0x070f] claimed by PIIX4 SMB
[    0.481987] pci 0000:00:02.0: [1af4:1000] type 00 class 0x020000
[    0.484413] pci 0000:00:02.0: reg 0x10: [io  0xc0c0-0xc0df]
[    0.486409] pci 0000:00:02.0: reg 0x14: [mem 0xfeb80000-0xfeb80fff]
[    0.491499] pci 0000:00:02.0: reg 0x20: [mem 0xfebf4000-0xfebf7fff 64bit pref]
[    0.494503] pci 0000:00:02.0: reg 0x30: [mem 0xfeb00000-0xfeb7ffff pref]
[    0.497857] pci 0000:00:03.0: [1af4:1009] type 00 class 0x000200
[    0.500299] pci 0000:00:03.0: reg 0x10: [io  0xc080-0xc0bf]
[    0.502059] pci 0000:00:03.0: reg 0x14: [mem 0xfeb81000-0xfeb81fff]
[    0.507286] pci 0000:00:03.0: reg 0x20: [mem 0xfebf8000-0xfebfbfff 64bit pref]
[    0.511396] pci 0000:00:04.0: [1af4:1001] type 00 class 0x010000
[    0.514499] pci 0000:00:04.0: reg 0x10: [io  0xc000-0xc07f]
[    0.516214] pci 0000:00:04.0: reg 0x14: [mem 0xfeb82000-0xfeb82fff]
[    0.520499] pci 0000:00:04.0: reg 0x20: [mem 0xfebfc000-0xfebfffff 64bit pref]
[    0.525082] ACPI: PCI Interrupt Link [LNKA] (IRQs 5 *10 11)
[    0.526632] ACPI: PCI Interrupt Link [LNKB] (IRQs 5 *10 11)
[    0.528627] ACPI: PCI Interrupt Link [LNKC] (IRQs 5 10 *11)
[    0.530623] ACPI: PCI Interrupt Link [LNKD] (IRQs 5 10 *11)
[    0.531571] ACPI: PCI Interrupt Link [LNKS] (IRQs *9)
[    0.533704] iommu: Default domain type: Translated 
[    0.535494] vgaarb: loaded
[    0.536295] SCSI subsystem initialized
[    0.536523] pps_core: LinuxPPS API ver. 1 registered
[    0.538507] pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
[    0.540502] PTP clock support registered
[    0.541502] EDAC MC: Ver: 3.0.0
[    0.543513] PCI: Using ACPI for IRQ routing
[    0.544814] NetLabel: Initializing
[    0.545498] NetLabel:  domain hash size = 128
[    0.546497] NetLabel:  protocols = UNLABELED CIPSOv4 CALIPSO
[    0.547506] NetLabel:  unlabeled traffic allowed by default
[    0.548529] hpet0: at MMIO 0xfed00000, IRQs 2, 8, 0
[    0.549496] hpet0: 3 comparators, 64-bit 100.000000 MHz counter
[    0.555552] clocksource: Switched to clocksource kvm-clock
[    0.623882] VFS: Disk quotas dquot_6.6.0
[    0.625021] VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
[    0.627113] pnp: PnP ACPI init
[    0.628219] pnp: PnP ACPI: found 5 devices
[    0.[635](https://github.com/cilium/cilium/actions/runs/8184258448/job/22378449602#step:8:645)562] thermal_sys: Registered thermal governor 'step_wise'
[    0.635563] thermal_sys: Registered thermal governor 'user_space'
[    0.641892] clocksource: acpi_pm: mask: 0xffffff max_cycles: 0xffffff, max_idle_ns: 2085701024 ns
[    0.645902] pci_bus 0000:00: resource 4 [io  0x0000-0x0cf7 window]
[    0.647559] pci_bus 0000:00: resource 5 [io  0x0d00-0xffff window]
[    0.649238] pci_bus 0000:00: resource 6 [mem 0x000a0000-0x000bffff window]
[    0.651069] pci_bus 0000:00: resource 7 [mem 0xc0000000-0xfebfffff window]
[    0.652894] pci_bus 0000:00: resource 8 [mem 0x340000000-0x3bfffffff window]
[    0.654809] NET: Registered protocol family 2
[    0.656239] IP idents hash table entries: 262144 (order: 9, 2097152 bytes, linear)
[    0.659098] tcp_listen_portaddr_hash hash table entries: 8192 (order: 5, 131072 bytes, linear)
[    0.661432] TCP established hash table entries: 131072 (order: 8, 1048576 bytes, linear)
[    0.6[637](https://github.com/cilium/cilium/actions/runs/8184258448/job/22378449602#step:8:647)67] TCP bind hash table entries: 65536 (order: 8, 1048576 bytes, linear)
[    0.666047] TCP: Hash tables configured (established 131072 bind 65536)
[    0.667836] UDP hash table entries: 8192 (order: 6, 262144 bytes, linear)
[    0.669721] UDP-Lite hash table entries: 8192 (order: 6, 262144 bytes, linear)
[    0.671690] NET: Registered protocol family 1
[    0.673344] RPC: Registered named UNIX socket transport module.
[    0.674977] RPC: Registered udp transport module.
[    0.676249] RPC: Registered tcp transport module.
[    0.677573] RPC: Registered tcp NFSv4.1 backchannel transport module.
[    0.679314] pci 0000:00:01.0: PIIX3: Enabling Passive Release
[    0.680903] pci 0000:00:00.0: Limiting direct PCI/PCI transfers
[    0.682605] pci 0000:00:01.0: Activating ISA DMA hang workarounds
[    0.684378] PCI: CLS 0 bytes, default 64
[    0.685552] Unpacking initramfs...
[    1.116[644](https://github.com/cilium/cilium/actions/runs/8184258448/job/22378449602#step:8:654)] Freeing initrd memory: 39040K
[    1.117821] PCI-DMA: Using software bounce buffering for IO (SWIOTLB)
[    1.119579] software IO TLB: mapped [mem 0xbbfdd000-0xbffdd000] (64MB)
[    1.122438] check: Scanning for low memory corruption every 60 seconds
[    1.124516] Initialise system trusted keyrings
[    1.125856] workingset: timestamp_bits=40 max_order=22 bucket_order=0
[    1.128978] NFS: Registering the id_resolver key type
[    1.130421] Key type id_resolver registered
[    1.131567] Key type id_legacy registered
[    1.132786] 9p: Installing v9fs 9p2000 file system support
[    1.136323] Key type asymmetric registered
[    1.137504] Asymmetric key parser 'x509' registered
[    1.138850] Block layer SCSI generic (bsg) driver version 0.4 loaded (major 250)
[    1.141063] io scheduler mq-deadline registered
[    1.142359] io scheduler kyber registered
[    1.143821] input: Power Button as /devices/LNXSYSTM:00/LNXPWRBN:00/input/input0
[    1.145927] ACPI: Power Button [PWRF]
[    1.159243] PCI Interrupt Link [LNKB] enabled at IRQ 10
[    1.174316] PCI Interrupt Link [LNKC] enabled at IRQ 11
[    1.189336] PCI Interrupt Link [LNKD] enabled at IRQ 11
[    1.192757] Serial: 8250/16550 driver, 4 ports, IRQ sharing enabled
[    1.220071] 00:03: ttyS0 at I/O 0x3f8 (irq = 4, base_baud = 115200) is a 16550A
[    1.223029] Non-volatile memory driver v1.3
[    1.226828] loop: module loaded
[    1.230960] virtio_blk virtio2: [vda] 41943040 512-byte logical blocks (21.5 GB/20.0 GiB)
[    1.235047] scsi host0: ata_piix
[    1.236156] scsi host1: ata_piix
[    1.236908] ata1: PATA max MWDMA2 cmd 0x1f0 ctl 0x3f6 bmdma 0xc0e0 irq 14
[    1.238504] ata2: PATA max MWDMA2 cmd 0x170 ctl 0x376 bmdma 0xc0e8 irq 15
[    1.242067] e100: Intel(R) PRO/100 Network Driver, 3.5.24-k2-NAPI
[    1.243468] e100: Copyright(c) 1999-2006 Intel Corporation
[    1.244739] e1000: Intel(R) PRO/1000 Network Driver - version 7.3.21-k8-NAPI
[    1.246337] e1000: Copyright (c) 1999-2006 Intel Corporation.
[    1.247622] e1000e: Intel(R) PRO/1000 Network Driver - 3.2.6-k
[    1.248932] e1000e: Copyright(c) 1999 - 2015 Intel Corporation.
[    1.250276] sky2: driver version 1.30
[    1.251146] i8042: PNP: PS/2 Controller [PNP0303:KBD,PNP0f13:MOU] at 0x60,0x64 irq 1,12
[    1.254083] serio: i8042 KBD port at 0x60,0x64 irq 1
[    1.255206] serio: i8042 AUX port at 0x60,0x64 irq 12
[    1.256704] rtc_cmos 00:04: RTC can wake from S4
[    1.258525] input: AT Translated Set 2 keyboard as /devices/platform/i8042/serio0/input/input1
[    1.260156] rtc_cmos 00:04: registered as rtc0
[    1.262086] rtc_cmos 00:04: alarms up to one day, y3k, 242 bytes nvram, hpet irqs
[    1.264873] IPVS: Registered protocols (TCP, UDP)
[    1.266563] IPVS: Connection hash table configured (size=4096, memory=32Kbytes)
[    1.269034] IPVS: ipvs loaded.
[    1.269901] IPVS: [rr] scheduler registered.
[    1.271189] NET: Registered protocol family 10
[    1.272708] Segment Routing with IPv6
[    1.273958] sit: IPv6, IPv4 and MPLS over IPv4 tunneling driver
[    1.275674] NET: Registered protocol family 17
[    1.276906] bridge: filtering via arp/ip/ip6tables is no longer available by default. Update your scripts to load br_netfilter if you need this.
[    1.280624] 9pnet: Installing 9P2000 support
[    1.283132] Key type dns_resolver registered
[    1.284841] IPI shorthand broadcast: enabled
[    1.286065] sched_clock: Marking stable (1108034932, 177992920)->(1320089343, -34061491)
[    1.288495] registered taskstats version 1
[    1.289667] Loading compiled-in X.509 certificates
[    1.291022] ima: No TPM chip found, activating TPM-bypass!
[    1.292519] ima: Allocated hash algorithm: sha1
[    1.293780] ima: No architecture policies found
[    1.295583] PM:   Magic number: 0:862:166
[    1.296727] printk: console [netcon0] enabled
[    1.297944] netconsole: network logging started
[    1.400499] Freeing unused kernel image memory: 1588K
[    1.401939] Write protecting the kernel read-only data: 22528k
[    1.403978] Freeing unused kernel image memory: 2028K
[    1.405664] Freeing unused kernel image memory: 1844K
[    1.407072] Run /init as init process
[    1.408617] mkdir (81) used greatest stack depth: 14520 bytes left
Loading, please wait...
[    1.411968] ln (91) used greatest stack depth: 14328 bytes left
Starting systemd-udevd version 255.3-2

[    1.460164] virtio_net virtio0 enp0s2: renamed from eth0
Begin: Loading essential drivers ... done.
Begin: Running /scripts/init-premount ... done.
Begin: Mounting root file system ... Begin: Running /scripts/local-top ... done.
Begin: Running /scripts/local-premount ... done.
[    1.487462] fstype (136) used greatest stack depth: 14312 bytes left
Warning: fsck not present, so skipping root file system
[    1.890670] input: ImExPS/2 Generic Explorer Mouse as /devices/platform/i8042/serio1/input/input3
[    4.096604] EXT4-fs (vda): mounted filesystem with ordered data mode. Opts: (null)
[    4.098899] mount (137) used greatest stack depth: 13976 bytes left
done.
Begin: Running /scripts/local-bottom ... done.
Begin: Running /scripts/init-bottom ... [    4.104128] (udev-worker) (122) used greatest stack depth: 13440 bytes left
done.
[    4.108289] run-init (147) used greatest stack depth: 13368 bytes left
SELinux:  Could not open policy file <= /etc/selinux/targeted/policy/policy.33:  No such file or directory
[    4.142623] systemd[1]: systemd 255.3-2 running in system mode (+PAM +AUDIT +SELINUX +APPARMOR +IMA +SMACK +SECCOMP +GCRYPT -GNUTLS +OPENSSL +ACL +BLKID +CURL +ELFUTILS +FIDO2 +IDN2 -IDN +IPTC +KMOD +LIBCRYPTSETUP +LIBFDISK +PCRE2 -PWQUALITY +P11KIT +QRENCODE +TPM2 +BZIP2 +LZ4 +XZ +ZLIB +ZSTD -BPF_FRAMEWORK -XKBCOMMON +UTMP +SYSVINIT default-hierarchy=unified)
[    4.151168] systemd[1]: Detected virtualization kvm.
[    4.152584] systemd[1]: Detected architecture x86-64.

Welcome to Debian GNU/Linux trixie/sid!


[    4.155667] systemd[1]: Hostname set to <kind-5.4>.
[    4.223420] systemd[1]: Queued start job for default target graphical.target.
[    4.233086] systemd[1]: Created slice system-getty.slice - Slice /system/getty.
[  OK  ] Created slice system-getty.slice - Slice /system/getty.

[    4.236684] systemd[1]: Created slice system-modprobe.slice - Slice /system/modprobe.
[  OK  ] Created slice system-modprobe.slice - Slice /system/modprobe.

[    4.240501] systemd[1]: Created slice system-serial\x2dgetty.slice - Slice /system/serial-getty.
[  OK  ] Created slice system-serial\x2dget…slice - Slice /system/serial-getty.

[    4.244620] systemd[1]: Created slice user.slice - User and Session Slice.
[  OK  ] Created slice user.slice - User and Session Slice.

[    4.247800] systemd[1]: Started systemd-ask-password-console.path - Dispatch Password Requests to Console Directory Watch.
[  OK  ] Started systemd-ask-password-conso…equests to Console Directory Watch.

[    4.252267] systemd[1]: Started systemd-ask-password-wall.path - Forward Password Requests to Wall Directory Watch.
[  OK  ] Started systemd-ask-password-wall.…d Requests to Wall Directory Watch.

[    4.256731] systemd[1]: Set up automount proc-sys-fs-binfmt_misc.automount - Arbitrary Executable File Formats File System Automount Point.
[  OK  ] Set up automount proc-sys-fs-binfm…ormats File System Automount Point.

[    4.261671] systemd[1]: Expecting device dev-ttyS0.device - /dev/ttyS0...
         Expecting device dev-ttyS0.device - /dev/ttyS0...

[    4.2[646](https://github.com/cilium/cilium/actions/runs/8184258448/job/22378449602#step:8:656)96] systemd[1]: Reached target cryptsetup.target - Local Encrypted Volumes.
[  OK  ] Reached target cryptsetup.target - Local Encrypted Volumes.

[    4.268150] systemd[1]: Reached target integritysetup.target - Local Integrity Protected Volumes.
[  OK  ] Reached target integritysetup.targ… Local Integrity Protected Volumes.

[    4.272093] systemd[1]: Reached target paths.target - Path Units.
[  OK  ] Reached target paths.target - Path Units.

[    4.274894] systemd[1]: Reached target remote-fs.target - Remote File Systems.
[  OK  ] Reached target remote-fs.target - Remote File Systems.

[    4.278348] systemd[1]: Reached target slices.target - Slice Units.
[  OK  ] Reached target slices.target - Slice Units.

[    4.281242] systemd[1]: Reached target swap.target - Swaps.
[  OK  ] Reached target swap.target - Swaps.

[    4.283828] systemd[1]: Reached target timers.target - Timer Units.
[  OK  ] Reached target timers.target - Timer Units.

[    4.286914] systemd[1]: Reached target veritysetup.target - Local Verity Protected Volumes.
[  OK  ] Reached target veritysetup.target - Local Verity Protected Volumes.

[    4.290887] systemd[1]: Listening on systemd-fsckd.socket - fsck to fsckd communication Socket.
[  OK  ] Listening on systemd-fsckd.socket …fsck to fsckd communication Socket.

[    4.295089] systemd[1]: Listening on systemd-initctl.socket - initctl Compatibility Named Pipe.
[  OK  ] Listening on systemd-initctl.socke…- initctl Compatibility Named Pipe.

[    4.299194] systemd[1]: Listening on systemd-journald-dev-log.socket - Journal Socket (/dev/log).
[  OK  ] Listening on systemd-journald-dev-…socket - Journal Socket (/dev/log).

[    4.303430] systemd[1]: Listening on systemd-journald.socket - Journal Socket.
[  OK  ] Listening on systemd-journald.socket - Journal Socket.

[    4.307029] systemd[1]: Listening on systemd-networkd.socket - Network Service Netlink Socket.
[  OK  ] Listening on systemd-networkd.socket - Network Service Netlink Socket.

[    4.310894] systemd[1]: systemd-pcrextend.socket - TPM2 PCR Extension (Varlink) was skipped because of an unmet condition check (ConditionSecurity=measured-uki).
[    4.314425] systemd[1]: Listening on systemd-udevd-control.socket - udev Control Socket.
[  OK  ] Listening on systemd-udevd-control.socket - udev Control Socket.

[    4.318177] systemd[1]: Listening on systemd-udevd-kernel.socket - udev Kernel Socket.
[  OK  ] Listening on systemd-udevd-kernel.socket - udev Kernel Socket.

[    4.331683] systemd[1]: Mounting dev-hugepages.mount - Huge Pages File System...
         Mounting dev-hugepages.mount - Huge Pages File System...

[    4.336521] systemd[1]: Mounting dev-mqueue.mount - POSIX Message Queue File System...
         Mounting dev-mqueue.mount - POSIX Message Queue File System...

[    4.342950] systemd[1]: Mounting sys-kernel-debug.mount - Kernel Debug File System...
         Mounting sys-kernel-debug.mount - Kernel Debug File System...

[    4.348342] systemd[1]: Mounting sys-kernel-tracing.mount - Kernel Trace File System...
         Mounting sys-kernel-tracing.mount - Kernel Trace File System...

[    4.353349] systemd[1]: kmod-static-nodes.service - Create List of Static Device Nodes was skipped because of an unmet condition check (ConditionFileNotEmpty=/lib/modules/5.4.268/modules.devname).
[    4.358584] systemd[1]: Starting modprobe@configfs.service - Load Kernel Module configfs...
         Starting modprobe@configfs.service - Load Kernel Module configfs...

[    4.363338] systemd[1]: Starting modprobe@dm_mod.service - Load Kernel Module dm_mod...
         Starting modprobe@dm_mod.service - Load Kernel Module dm_mod...

[    4.368080] systemd[1]: Starting modprobe@drm.service - Load Kernel Module drm...
         Starting modpr[    4.370798] systemd[1]: Starting modprobe@efi_pstore.service - Load Kernel Module efi_pstore...
obe@drm.service - Load Kernel Module drm...
[    4.375323] systemd[1]: Starting modprobe@fuse.service - Load Kernel Module fuse...

         Starting modprobe@efi_pstore[    4.379106] systemd[1]: Starting modprobe@loop.service - Load Kernel Module loop...
.servi… - Load Kernel Modu[    4.382572] systemd[1]: systemd-fsck-root.service - File System Check on Root Device was skipped because of an unmet condition check (ConditionPathIsReadWrite=!/).
le efi_pstore...

         Starting modprobe@fuse.service - Load Kernel Module fuse...

         Starting modprobe@loop.service - Load Kernel Module loop...

[    4.391234] systemd[1]: Starting systemd-journald.service - Journal Service...
         Starting systemd-journald.service - Journal Service...

[    4.395895] systemd[1]: Starting systemd-modules-load.service - Load Kernel Modules...
         Starting systemd-modules-load.service - Load Kernel Modules...

[    4.401027] systemd[1]: Starting systemd-network-generator.service - Generate network units from Kernel command line...
         Starting systemd-network-generator…k units from Kernel command line...

[    4.407508] systemd[1]: systemd-pcrmachine.service - TPM2 PCR Machine ID Measurement was skipped because of an unmet condition check (ConditionSecurity=measured-uki).
[    4.413719] systemd[1]: Starting systemd-remount-fs.service - Remount Root and Kernel File Systems...
         Starting systemd-remount-fs.servic…unt Root and Kernel File Systems...

[    4.419348] systemd[1]: Starting systemd-tmpfiles-setup-dev-early.service - Create Static Device Nodes in /dev gracefully...
         Starting systemd-tmpfiles-setup-de… Device Nodes in /dev gracefully...

[    4.426671] systemd[1]: systemd-tpm2-setup-early.service - TPM2 SRK Setup (Early) was skipped because of an unmet condition check (ConditionSecurity=measured-uki).
[    4.430122] systemd-journald[176]: Collecting audit messages is disabled.
[    4.431959] systemd[1]: Starting systemd-udev-trigger.service - Coldplug All udev Devices...
[    4.432162] EXT4-fs (vda): re-mounted. Opts: errors=remount-ro
         Starting systemd-udev-trigger.service - Coldplug All udev Devices...

[    4.443887] systemd[1]: Mounted dev-hugepages.mount - Huge Pages File System.
[  OK  ] Mounted dev-hugepages.mount - Huge Pages File System.

[    4.448514] systemd[1]: Mounted dev-mqueue.mount - POSIX Message Queue File System.
[  OK  ] Mounted dev-mqueue.mount - POSIX Message Queue File System.

[    4.452670] systemd[1]: Mounted sys-kernel-debug.mount - Kernel Debug File System.
[  OK  ] Mounted sys-kernel-debug.mount - Kernel Debug File System.

[    4.456768] systemd[1]: Mounted sys-kernel-tracing.mount - Kernel Trace File System.
[  OK  ] Mounted sys-kernel-tracing.mount - Kernel Trace File System.

[    4.460906] systemd[1]: Started systemd-journald.service - Journal Service.
[  OK  ] Started systemd-journald.service - Journal Service.

[  OK  ] Finished modprobe@configfs.service - Load Kernel Module configfs.

[  OK  ] Finished modprobe@dm_mod.service - Load Kernel Module dm_mod.

[  OK  ] Finished modprobe@drm.service - Load Kernel Module drm.

[  OK  ] Finished modprobe@efi_pstore.service - Load Kernel Module efi_pstore.

[  OK  ] Finished modprobe@fuse.service - Load Kernel Module fuse.

[  OK  ] Finished modprobe@loop.service - Load Kernel Module loop.

[  OK  ] Finished systemd-modules-load.service - Load Kernel Modules.

[  OK  ] Finished systemd-network-generator…ork units from Kernel command line.

[  OK  ] Finished systemd-remount-fs.servic…mount Root and Kernel File Systems.

[  OK  ] Finished systemd-tmpfiles-setup-de…ic Device Nodes in /dev gracefully.

[  OK  ] Finished systemd-udev-trigger.service - Coldplug All udev Devices.

[  OK  ] Reached target network-pre.target - Preparation for Network.

         Starting ifupdown-pre.service - He…synchronize boot up for ifupdown...

         Starting systemd-journal-flush.ser…sh Journal to Persistent Storage...

         Starting systemd-random-seed.service - Load/Save OS Random Seed...

[    4.516481] systemd-journald[176]: Received client request to flush runtime journal.
         Starting systemd-sysctl.service - Apply Kernel Variables...

         Starting systemd-sysusers.service - Create System Users...

[  OK  ] Finished ifupdown-pre.service - He…o synchronize boot up for ifupdown.

[  OK  ] Finished systemd-random-seed.service - Load/Save OS Random Seed.

[  OK  ] Finished systemd-sysctl.service - Apply Kernel Variables.

[  OK  ] Finished systemd-journal-flush.ser…lush Journal to Persistent Storage.

[  OK  ] Finished systemd-sysusers.service - Create System Users.

         Starting systemd-tmpfiles-setup-de…eate Static Device Nodes in /dev...

[  OK  ] Finished systemd-tmpfiles-setup-de…Create Static Device Nodes in /dev.

[  OK  ] Reached target local-fs-pre.target…Preparation for Local File Systems.

         Mounting host.mount - /host...

         Starting systemd-udevd.service - R…ager for Device Events and Files...

[  OK  ] Mounted host.mount - /host.

[  OK  ] Reached target local-fs.target - Local File Systems.

[  OK  ] Listening on systemd-sysext.socket…tension Image Management (Varlink).

         Starting ldconfig.service - Rebuild Dynamic Linker Cache...

         Starting networking.service - Raise network interfaces...

         Starting systemd-tmpfiles-setup.se…e Volatile Files and Directories...

[  OK  ] Finished systemd-tmpfiles-setup.se…ate Volatile Files and Directories.

         Starting systemd-journal-catalog-u…ervice - Rebuild Journal Catalog...

         Starting systemd-resolved.service - Network Name Resolution...

         Starting systemd-update-utmp.servi…ord System Boot/Shutdown in UTMP...

[  OK  ] Finished ldconfig.service - Rebuild Dynamic Linker Cache.

[  OK  ] Finished systemd-journal-catalog-u….service - Rebuild Journal Catalog.

[  OK  ] Started systemd-udevd.service - Ru…anager for Device Events and Files.

[  OK  ] Finished systemd-update-utmp.servi…ecord System Boot/Shutdown in UTMP.

[  OK  ] Finished networking.service - Raise network interfaces.

         Starting systemd-networkd.service - Network Configuration...

         Starting systemd-update-done.service - Update is Completed...

[  OK  ] Found device dev-ttyS0.device - /dev/ttyS0.

[  OK  ] Finished systemd-update-done.service - Update is Completed.

[  OK  ] Started systemd-resolved.service - Network Name Resolution.

[  OK  ] Reached target nss-lookup.target - Host and Network Name Lookups.

[  OK  ] Reached target sysinit.target - System Initialization.

[  OK  ] Listening on dbus.socket - D-Bus System Message Bus Socket.

         Starting docker.socket - Docker Socket for the API...

[  OK  ] Started systemd-networkd.service - Network Configuration.

[  OK  ] Listening on docker.socket - Docker Socket for the API.

[  OK  ] Reached target network.target - Network.

[  OK  ] Reached target sockets.target - Socket Units.

[  OK  ] Reached target basic.target - Basic System.

         Starting containerd.service - containerd container runtime...

[  OK  ] Started cron.service - Regular background program processing daemon.

         Starting dbus.service - D-Bus System Message Bus...

         Starting e2scrub_reap.service - Re…ne ext4 Metadata Check Snapshots...

         Starting ssh.service - OpenBSD Secure Shell server...

         Starting systemd-logind.service - User Login Management...

         Starting systemd-networkd-wait-onl…ait for Network to be Configured...

         Starting systemd-user-sessions.service - Permit User Sessions...

[  OK  ] Started dbus.service - D-Bus System Message Bus.

[  OK  ] Finished systemd-user-sessions.service - Permit User Sessions.

[  OK  ] Started getty@tty1.service - Getty on tty1.

[  OK  ] Started serial-getty@ttyS0.service - Serial Getty on ttyS0.

[  OK  ] Reached target getty.target - Login Prompts.

[  OK      4.854[647](https://github.com/cilium/cilium/actions/runs/8184258448/job/22378449602#step:8:657)] e2scrub_all (321) used greatest stack depth: 13200 bytes left
0m] Started ssh.service - OpenBSD Secure Shell server.

[  OK  ] Finished e2scrub_reap.service - Re…line ext4 Metadata Check Snapshots.

[  OK  ] Started systemd-logind.service - User Login Management.

[  OK  ] Started containerd.service - containerd container runtime.

[  OK  ] Finished systemd-networkd-wait-onl… Wait for Network to be Configured.

[  OK  ] Reached target network-online.target - Network is Online.

         Starting docker.service - Docker Application Container Engine...

[    6.433060] Bridge firewalling registered
[    6.484[741](https://github.com/cilium/cilium/actions/runs/8184258448/job/22378449602#step:8:751)] Initializing XFRM netlink socket
[  OK  ] Started docker.service - Docker Application Container Engine.

[  OK  ] Reached target multi-user.target - Multi-User System.

[  OK  ] Reached target graphical.target - Graphical Interface.

         Starting systemd-update-utmp-runle…- Record Runlevel Change in UTMP...

[  OK  ] Finished systemd-update-utmp-runle…e - Record Runlevel Change in UTMP.



Debian GNU/Linux trixie/sid kind-5.4 ttyS0

kind-5 login: 
Error: Process completed with exit code 41.
```

Example run: https://github.com/cilium/cilium/actions/runs/8184258448/job/22378449602

--- a/.github/actions/lvh-kind/action.yaml
+++ b/.github/actions/lvh-kind/action.yaml
@@ -28,6 +28,7 @@ runs:
         mem: 12G
         install-dependencies: 'true'
         port-forward: '6443:6443'
+        ssh-startup-wait-retries: 600
         cmd: |
           git config --global --add safe.directory /host
 

